### ISSUE TITLE
Memory leak detection

### ISSUE THREAD

**Issue by gaejuck** [Cruze03/discord-utilities#20]
I think memory was leaked due to this error. 
Is there any way to fix it?
``
L 06/13/2021 - 14:39:32: [discord_api.smx] [DISCORD] Couldn't Send Webhook - Fail 0 400
L 06/13/2021 - 14:40:25: [discord_api.smx] [DISCORD] Couldn't Send Webhook - Fail 0 400
``
``
L 06/13/2021 - 14:47:04: [SM] MEMORY LEAK DETECTED IN PLUGIN (file "discord_api.smx")
L 06/13/2021 - 14:47:04: [SM] Unloading plugin to free 29259 handles.
L 06/13/2021 - 14:47:04: [SM] Contact the author(s) of this plugin to correct this error.
L 06/13/2021 - 14:47:04: --------------------------------------------------------------------------
L 06/13/2021 - 14:47:04: Type	DataPack            |	Count	334
L 06/13/2021 - 14:47:04: Type	PrivateFwd          |	Count	1
L 06/13/2021 - 14:47:04: Type	Trie                |	Count	3
L 06/13/2021 - 14:47:04: Type	HTTPHandle          |	Count	169
L 06/13/2021 - 14:47:04: Type	JanssonObject       |	Count	28752
L 06/13/2021 - 14:47:04: --------------------------------------------------------------------------
L 06/13/2021 - 14:47:04: Oldest Living Handle: Trie created at 06/13/2021 - 14:01:12
L 06/13/2021 - 14:47:04: Newest Living Handle: JanssonObject created at 06/13/2021 - 14:47:04
L 06/13/2021 - 14:47:04: -- Approximately 27941 bytes of memory are in use by (29259) Handles.
``

**Comment by crashzk** [comment]
Leak would be in the [Discord API](https://github.com/Cruze03/sourcemod-discord), not the **Discord Utilities**

**Comment by gaejuck** [comment]
yeah I know, but I wrote here because I am using sourcemod-discord that you have worked on and I cannot put an issue on your sourcemod-discord repository.

**Comment by gaejuck** [comment]
L 06/23/2021 - 22:32:27: [SM] Exception reported: Invalid <Object> handle b1800d6 (error 3)
L 06/23/2021 - 22:32:27: [SM] Blaming: discord_api.smx
L 06/23/2021 - 22:32:27: [SM] Call stack trace:
L 06/23/2021 - 22:32:27: [SM]   [0] json_object_get
L 06/23/2021 - 22:32:27: [SM]   [1] Line 18, C:\Users\STARS\Desktop\Coding\Discord Utilities\scripting\include\discord/stocks.inc::JsonObjectGetString
L 06/23/2021 - 22:32:27: [SM]   [2] Line 112, C:\Users\STARS\Desktop\Coding\Discord Utilities\scripting\discord_api.sp::BuildAuthHeader
L 06/23/2021 - 22:32:27: [SM]   [3] Line 135, C:\Users\STARS\Desktop\Coding\Discord Utilities\scripting\discord_api.sp::PrepareRequest
L 06/23/2021 - 22:32:27: [SM]   [4] Line 21, discord/GetGuilds.sp::ThisSendRequest
L 06/23/2021 - 22:32:27: [SM]   [5] Line 46, discord/GetGuilds.sp::GetGuildsData

**Comment by justkamiii** [comment]
this is also happening to me
